src/widgets/helper/ChannelView.cpp

```diff
@@ -1658,37 +1659,94 @@ void ChannelView::mouseMoveEvent(QMouseEvent *event)
     auto element = &hoverLayoutElement->getCreator();
     bool isLinkValid = hoverLayoutElement->getLink().isValid();
     auto emoteElement = dynamic_cast<const EmoteElement *>(element);
+    auto layeredEmoteElement =
+        dynamic_cast<const LayeredEmoteElement *>(element);
+    bool isNotEmote = emoteElement == nullptr && layeredEmoteElement == nullptr;
 
     if (element->getTooltip().isEmpty() ||
-        (isLinkValid && emoteElement == nullptr &&
-         !getSettings()->linkInfoTooltip))
+        (isLinkValid && isNotEmote && !getSettings()->linkInfoTooltip))
     {
         tooltipWidget->hide();
     }
     else
     {
         auto badgeElement = dynamic_cast<const BadgeElement *>(element);
 
-        if ((badgeElement || emoteElement) &&
+        if ((badgeElement || emoteElement || layeredEmoteElement) &&
             getSettings()->emotesTooltipPreview.getValue())
         {
             if (event->modifiers() == Qt::ShiftModifier ||
                 getSettings()->emotesTooltipPreview.getValue() == 1)
             {
                 if (emoteElement)
                 {
-                    tooltipWidget->setImage(
-                        emoteElement->getEmote()->images.getImage(3.0));
+                    tooltipWidget->setOne({
+                        emoteElement->getEmote()->images.getImage(3.0),
+                        element->getTooltip(),
+                    });
+                }
+                else if (layeredEmoteElement)
+                {
+                    auto &layeredEmotes = layeredEmoteElement->getEmotes();
+                    // Should never be empty but ensure it
+                    if (!layeredEmotes.empty())
+                    {
+                        std::vector<TooltipEntry> records;
```

Comment:
We call many of the variables that have to do with TooltipEntries `record` or `records`, I would've called them `entry` and `entries`

Reply:
Good catch -- while I was developing this PR I changed the name from `TooltipRecord` to `TooltipEntry` but forgot to update variable names